src/pheromones/src/HiveConnectNetworkConfigSetRequestDTO.cpp

```diff
@@ -0,0 +1,83 @@
+#include "HiveConnectNetworkConfigSetRequestDTO.h"
+
+HiveConnectNetworkConfigSetRequestDTO::HiveConnectNetworkConfigSetRequestDTO() {
+    m_networkAccess = std::nullopt;
+    m_rootNode = std::nullopt;
+    m_meshEnable = std::nullopt;
```

Comment:
no need for nullopt, though it's more explicit, the default constructor is already nullopt. Also you should use member initialization if you want to be explicit.